src/client/controllers/LoadTestController.ts

```diff
@@ -0,0 +1,58 @@
+import store from "../toolkit-refactor/store";
+import { appDispatch } from "../toolkit-refactor/store";
+import { responseDataSaved, reqResUpdated } from "../toolkit-refactor/reqRes/reqResSlice";
+import { ReqRes, WindowExt } from "../../types";
+import { LoadTestResult } from "../components/main/loadTest/LoadTest";
+import { graphUpdated } from "../toolkit-refactor/graphPoints/graphPointsSlice";
+
+const { api } = window as unknown as WindowExt;
+
+const LoadTestController = {
+
+  convertLoadTestResultToEventsArray(loadTestResult: LoadTestResult): Record<string, unknown>[] {
+    return [
+      { totalSent: loadTestResult.totalSent },
+      { totalReceived: loadTestResult.totalReceived },
+      { totalMissed: loadTestResult.totalMissed },
+      { averageResponseTime: loadTestResult.averageResponseTime },
+      { totalNotSent: loadTestResult.totalNotSent },
+      { errorCounts: loadTestResult.errorCounts },
+    ];
+  },
```

Comment:
Nit: perhaps let's use destructuring assignments?

```suggestion
  convertLoadTestResultToEventsArray(loadTestResult: LoadTestResult): Record<string, unknown>[] {
    const {totalSent, totalReceived.....} = loadTestResult;
    return [
      { totalSent },
      { totalReceived },
      { totalMissed },
      { averageResponseTime },
      { totalNotSent },
      { errorCounts },
    ];
  },
```